AprilSylph/XKit-Rewritten
# Warn about unloaded/updated extension errors in Chromium

### Description


Adds a line to the "extension context invalidated" error message shown when Chromium has autoupdated the extension but the current tab has not been refreshed since the autoupdate. (Unlike Firefox tabs, Chromium tabs in this state act as though the extension has simply been disabled.)

I would rather have something like #451, but that would necessitate more discussion over whether we want to do that and, as a simplest possible solution, a setInterval that checks for the property. If we always imported storage calls from a util I'd put it there, but we don't.

### Testing steps


a)

- Load the extension in a Chromium browser and enable Quick Tags.
- Disable the extension without refreshing the tab, then reenable it (simulating an extension update)
- Draft a post with Quick Reblog and confirm that the "extension context invalidated" error message has the additional line.

b)

- Unrevert the test commit to cause a network error.
- Draft a post with Quick Reblog and confirm that the error message does not have the additional line.

c)

- Perform the first test in Firefox and confirm that there is no error; Firefox does extension upgrades within active tabs.
- Perform the second test in Firefox and confirm the same result.

d)

- Perform these tests with this PR merged into the MV3 branch.
